app/web/src/router.ts

```diff
@@ -32,6 +32,11 @@ const routes: RouteRecordRaw[] = [
       };
     },
     children: [
+      {
```

Comment:
love this endpoint, excited to see how the viz can help us debug and optimize stuff in the future